hypervisor/src/kvm/mod.rs

```diff
@@ -1076,6 +1076,16 @@ impl hypervisor::Hypervisor for KvmHypervisor {
 
         Ok(data)
     }
+
+    ///
+    /// Get the number of supported hardware breakpoints
+    ///
+    fn get_guest_debug_hw_bps(&self) -> i32 {
```

Comment:
The number of breakpoints cannot go negative so it is better to use `usize`.

Reply:
Good point.